guides/continuous-integration.md

```diff
@@ -1,5 +1,24 @@
-# CI/CD
+# Continuous integration & delivery
 
-CI/CD
+CI/CD is used across all projects at the Data Shed as a way of streamlining
+deployments, automatically testing code and protecting failing tests from
+bypassing merge pipelines.
+
+The variety of work that comes through the Shed varies from project to project.
+Having clients with different tech stacks and subscriptions to various providers
+means not every pipeline looks the same. The Data Shed still applies the same
+general set of functional tooling across projects even though the tools
+themselves may differ in implementation. The Shed seeks to automate version
+control, packaging, testing, formatting, and linting code. The Data Shed makes
+heavy use of [terraform](../standards/terraform-standards.md)
+
+Tools used in the past:
```

Comment:
Notable omission: Azure DevOps (formerly VSTS, formerly TFS).

Less notable omission, and not one I'd like to admit much publicly, but we've had a few clients that have used Jenkins.